template/cordova/lib/run.js

```diff
@@ -39,7 +39,7 @@ module.exports.run = function (options) {
 
     // parse arg
     var args  = nopt({
-        'archs': String,
+        'archs': [String],
```

Comment:
Update: in terms of nopt `String` argument type is the same as `[String]`. The latter syntax (with array literal) just allows to specify multiple allowed types